src/core/builtins/builtins_ssl.ml

```diff
@@ -139,12 +152,27 @@ let _ =
         Lang.nullable_t Lang.string_t,
         Some Lang.null,
         Some "SSL certificate password" );
-      ( "protocol",
+      ( "min_protocol",
+        Lang.string_t,
+        Some (Lang.string "ssl.3"),
+        Some
+          "Minimal accepted SSL protocol. One of, from least recent to most \
+           recent: `\"ssl.3\"`, `\"tls.1\"`, `\"tls.1.1\"`, `\"tls.1.2\"` or \
+           `\"tls.1.3\"`. The most recent available protocol between client \
+           and server is negotiated when initiating communication between \
+           minimal and maximal protocol version. All protocols up to \
+           `\"tls.1.2\"` and above are now deprecated so you might want to set \
+           this value to one of those two." );
+      ( "max_protocol",
         Lang.string_t,
         Some (Lang.string "tls.1.3"),
         Some
-          {|Accepted SSL protocol. One of: `"ssl.2.3"`, `"ssl.3"`, `"tls.1"`, `"tls.1.1"`, `"tls.1.2"` or `"tls.1.3"`|}
-      );
+          "Maximal accepted SSL protocol. One of, from least recent to most \
+           recent: `\"ssl.2.3\"`, `\"ssl.3\"`, `\"tls.1\"`, `\"tls.1.1\"`, \
```

Comment:
`ssl.2.3` is missing everywhere.

Reply:
It's not accepted by these functions. I've removed it from the doc.